guarddog/analyzer/sourcecode/__init__.py

```diff
@@ -14,17 +15,17 @@
 )
 
 SOURCECODE_RULES = {
-    ECOSYSTEM.PYPI: set(),
-    ECOSYSTEM.NPM: set()
-}  # type: dict[ECOSYSTEM, set[str]]
+    ECOSYSTEM.PYPI: list(),
```

Comment:
not: that's because dictionary is unhashable now that append whole rules instead of just the rule ID